excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/HiddenFieldsValidator.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.definition.store.excel.validation;
+
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.MapperException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionSheet;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.SheetName;
+
+import java.util.Map;
+import java.util.Optional;
+
+public class HiddenFieldsValidator {
+    Boolean retainHiddenValue;
+
+    public Boolean parseHiddenFields(DefinitionDataItem definitionDataItem, Map<String, DefinitionSheet> definitionSheets) {
+        final DefinitionSheet caseEventToFields = definitionSheets.get(SheetName.CASE_EVENT_TO_FIELDS.getName());
+        final DefinitionSheet caseFields = definitionSheets.get(SheetName.CASE_FIELD.getName());
+        Optional<DefinitionDataItem> caseField =
+            caseFields.getDataItems().stream().filter(caseFieldDataItem ->
+                definitionDataItem.getId().equals(caseFieldDataItem.getString(ColumnName.FIELD_TYPE))).findFirst();
```

Comment:
Is this correct that it's trying to find a case field by comparing an ID to the field type?

Reply:
Ignore above - I think this is for complex fields? Worth renaming the method to signify that?